src/routes/Settings/Settings.js

```diff
@@ -354,10 +391,11 @@ const Settings = () => {
                             <div className={styles['option-name-container']}>
                                 <div className={styles['label']}>Url</div>
                             </div>
-                            <div className={classnames(styles['option-input-container'], styles['info-container'], styles['selectable'])}>
-                                <div className={styles['label']}>
-                                    {profile.settings.streaming_server_url}
-                                </div>
+                            <div className={classnames(styles['option-input-container'], styles['edit-container'])}>
+                                <div className={styles['label']}>{profile.settings.streaming_server_url}</div>
+                                <Button className={classnames(styles['option-input-container'], styles['button-container'])} title={'Edit'} onClick={openEditServerUrlModal}>
```

Comment:
this is not a good place for reuse settings grid class names
use `configure-button-container` class name and put all of its styles there: flex, display, align-items, justify-content, background-color, width&height: 3.5rem etc